plugins/semgrep/semgrep.go

```diff
@@ -16,6 +16,19 @@ type ScannerSemgrep struct {
 	logger hclog.Logger
 }
 
+func getDefaultConfig() string {
+
+	if os.Getenv("CI") == "true" {
```

Comment:
Let's follow here scanio mask for env variables. 
CI move to SCANIO_MODE_CI or SCANIO_MODE=CI

SCANIO_MODE could be useful for the future using.

Reply:
I used "CI" env, because it's default value, that is set by github runner. Check logs of "Scanio analyze" step here: https://github.com/scan-io-git/scan-io/actions/runs/5867958254/job/15909774554.

So scanio can be run in ci and out of ci with same command. But depending on environment, it will automatically understands, what configs to use, and what workflow to use.

I can create isCI func in shared module, and make decision based on multiple variable, like `CI=true` and `SCANIO_MODE=CI`.

Wdyt?

Reply:
Oh I see. If it depends on Github CI it's doesn't matter. 
 My concern here is an intersection of the variable name with other possible tools and env variables with the same name  in other VCS. 

Taking into account the above you don't need to change something here